spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -380,11 +380,13 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
 
   override def visitDescribeDeltaHistory(
       ctx: DescribeDeltaHistoryContext): LogicalPlan = withOrigin(ctx) {
-    DescribeDeltaHistoryCommand(
-      Option(ctx.path).map(string),
-      Option(ctx.table).map(visitTableIdentifier),
-      Option(ctx.limit).map(_.getText.toInt),
-      Map.empty)
+    DescribeDeltaHistory(
```

Comment:
we can create the DescribeDeltaHistoryCommand here directly like before, and then assign the unresolved child operator to it for the analyzer to consume (as the child of a LeafRunnableCommand).